### PR TITLE
Airtable v2 source connector

### PR DESCRIPTION
**Pull request by Paul-Cornell**
See:

- API: https://unstructured-53-docs-74-airtable.mintlify.app/api-reference/ingest/source-connectors/airtable
- Open source: https://unstructured-53-docs-74-airtable.mintlify.app/open-source/ingest/source-connectors/airtable

### CODE REVIEW

File: api-reference/ingest/source-connectors/airtable.mdx [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
+<NewDocument />
+
+import SharedContentAirtable from '/snippets/sc-shared-text/airtable-cli-api.mdx';
+import SharedAPIKeyURL from '/snippets/general-shared-text/api-key-url.mdx';
 
 <SharedContentAirtable/>
+<SharedAPIKeyURL/>
 
-Finally, make sure to set the `--partition-by-api` flag and pass in your API key with `--api-key`:
+Now call the Unstructured CLI or Python SDK. The source connector can be any of the ones supported. This example uses the local source connector:
```
Comment (MKhalusova) [inline api-reference/ingest/source-connectors/airtable.mdx:15]: You mean the destination can be any of the ones supported? 
We're using Airtable here as a source.

File: open-source/ingest/source-connectors/airtable.mdx [hunk 2]
```diff
@@ -2,21 +2,28 @@
...
-import SharedContentAirtable from '/snippets/sc-shared-text/airtable.mdx';
+import NewDocument from '/snippets/general-shared-text/new-document.mdx';
+
+<NewDocument />
+
+import SharedContentAirtable from '/snippets/sc-shared-text/airtable-cli-api.mdx';
 
 <SharedContentAirtable/>
 
+Now call the Unstructured CLI or Python. The source connector can be any of the ones supported. This example uses the local source connector.
```
Comment (MKhalusova) [inline open-source/ingest/source-connectors/airtable.mdx:13]: Same here, the destination can be any of the ones supported. We're using Airtable as a source.

File: snippets/general-shared-text/airtable.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+The Airtable connector prerequisites:
```
Comment (MKhalusova) [inline snippets/general-shared-text/airtable.mdx:1]: This text doesn't show up on the staged pages for some reason.

**Review by MKhalusova — CHANGES_REQUESTED** [review]
A couple of small changes needed, please see the comments.

File: api-reference/ingest/source-connectors/airtable.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply api-reference/ingest/source-connectors/airtable.mdx:15]: Good catch! A copy-paste error. In fact, I went through and noticed this error in several other files. Fixing those now as well.

File: snippets/general-shared-text/airtable.mdx (hunk 3 above)
Comment (Paul-Cornell) [reply snippets/general-shared-text/airtable.mdx:1]: Looks like a Mintlify GitHub build timing issue. I can see it fine in my local build.

**Review by MKhalusova — APPROVED** [review]

**Pull request merged**